File: src/main/kotlin/com/nuutrai/reactor/listeners/PlayerDrop.java
package com.nuutrai.reactor.listeners;

import org.bukkit.event.EventHandler;
import org.bukkit.event.Listener;
import org.bukkit.event.player.PlayerDropItemEvent;

public class PlayerDrop implements Listener {

    @EventHandler
    public void onItemDrop(PlayerDropItemEvent e) {
        e.setCancelled(true);
    }

}

File: src/main/kotlin/com/nuutrai/reactor/listeners/PlayerDrop.kt
package com.nuutrai.reactor.listeners

import org.bukkit.event.EventHandler
import org.bukkit.event.Listener
import org.bukkit.event.player.PlayerDropItemEvent

class PlayerDrop : Listener {
	@EventHandler
	fun onItemDrop(e: PlayerDropItemEvent) {
		e.isCancelled = true
	}
}
